contracts/message-based/PriceAwareUpgradeable.sol

```diff
@@ -36,7 +36,19 @@ contract PriceAwareUpgradeable is OwnableUpgradeable {
     }
 
 
+    function getPricesFromMsg(bytes32[] memory symbols) internal view returns (uint256[] memory) {
```

Comment:
I don't think this method is necessary. Let's simpley rename _getPricesFromMsg to getPricesFromMsg and call it directly. We'll simplify the code and reduce gas costs